apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -290,22 +290,19 @@ export const EventSetupTab = (
                   setDefaultDuration(
                     selectedMultipleDuration.find((opt) => opt.value === option?.value) ?? null
                   );
-                  formMethods.setValue("length", Number(option?.value));
+                  formMethods.setValue("length", option?.value);
```

Comment:
It was already a number here, because it is picked from options[number].value which is a number.